packages/parsley/parsley.0.0.1/opam

```diff
@@ -0,0 +1,28 @@
+url {
+  src: "https://github.com/ghilesZ/parsley/archive/0.0.1.tar.gz"
+  checksum: [
+    "md5=69555b0223825683cbf697c3dc05965b"
+    "sha512=49abbdebf2c7fd55d198adf11f91965049126bf9dc9acd89bd170bf9ae8b264c68a46281d02e6ab8009ad1bc2c96b27c4313186a3693ca74e798abe3a1c7906b"
+  ]
+}
+opam-version: "2.0"
+maintainer: "Ghiles Ziat <ghiles.ziat@lip6.fr>"
+authors: [
+  "Ghiles Ziat <ghiles.ziat@lip6.fr>"
+]
+homepage: "https://github.com/ghilesZ/parsley"
+bug-reports: "https://github.com/ghilesZ/parsley/issues"
+dev-repo: "git+https://github.com/ghilesZ/parsley"
+license: "MIT"
+build: [
+  ["dune" "subst"] {pinned}
+  ["dune" "build" "-p" name "-j" jobs]
+]
+
+depends: [
+  "dune" {build}
```

Comment:
```suggestion
  "dune" {>= "1.11"}
```
Dune should not be tagged as a build dependency (see #14266) and the `dune-project` file in your repository says your package requires at least dune 1.11

Reply:
Thanks a lot, however the fix to this comment has been reverted, could you fix that again?

Reply:
I'm really sorry but this comment is still valid. Please don't forget to return it to your opam file upstream everytime you do a change here.